assets/src/legacy/view.js

```diff
@@ -44,11 +44,12 @@ var searchProjects = function(){
     // Get unique keywords for visible projects
     var getVisibleProjectsKeywords = function() {
         var keywordList = [];
-        var selector = '.liz-repository-project-item :visible .keywordList';
+        var selector = '.liz-repository-project-item :visible .liz-project';
 
         $(selector).each(function () {
-            if ($(this).text() !== '') {
-                var keywordsSplitByComma = $(this).text().toUpperCase().split(', ');
+            const keywords = $(this).attr('data-lizmap-keywords');
+            if (keywords !== '') {
+                var keywordsSplitByComma = keywords.toUpperCase().split(', ');
```

Comment:
* I would split with simple comma `,` and not with comma + space `, ` 
* Afterwards, I would trim with `keywordsSplitByComma[index].trim().split('/');`

Reply:
I know this is historical, but let's use this PR to improve it a bit